### ISSUE TITLE
ENHANCEMENT: Enhance footer section spacing, icons size & relocate translator from footer to navbar (header section) for easier accessibility.

### ISSUE THREAD

**Issue by shivanjalikarande** [AlfiyaSiddique/TastyTrails#267]
### Is there an existing issue raised for this?

- [X] I have searched the existing issues

### Describe the improvement

Footer section can be enhance by correct spacing between current elements & to increase appropriate size of social media icons. Translator should be present in navbar in upper right corner for easy accessibility to new user.

### Component

https://delightful-daifuku-a9f6ea.netlify.app/

### It should be implemented because

Footer should look good with correct spacing between elements. Social media icons should be easily navigate to correct sites, so they should be of large size than current. Translator should be present on area where user first look for, if is present in footer user will not easily find it in firstly, so it should be present in navbar with icon/text.

### Would you like to work on this issue?

Yes

### Terms & Conditions

- [X] I agree to follow the Code of Conduct
- [X] I'm a GSSOC'24 Ext contributor

**Comment by AlfiyaSiddique** [comment]
@shivanjalikarande change related to footer design and layout is being discussed here: #15 
If you are interested, please feel free to comment on issue 15 below. It will get assigned to you if there is no activity from the current assigner.
